File: lib/ce2/models/question_set.rb
class QuestionSet < Tuple
  attribute :name, :string
  attribute :info, :string
  attribute :explanation, :string
  attribute :position, :integer
  attribute :subtrack_id, :string
  attribute :fixed_info_width, :boolean
  attribute :deleted_at, :datetime
  attribute :published_at, :datetime

  # has_many :questions
  relates_to_many :questions do
    Question.where(Question.question_set_id.eq(id))
  end
end

File: public/javascript/app/models/question_set.js
June.define_set("QuestionSet", function(c) { with(c) {
  attributes({
    id: "string",
    name: "string",
    info: "string",
    explanation: "string",
    position: "integer",
    subtrack_id: "string",
    fixed_info_width: "boolean",
    deleted_at: "datetime",
    published_at: "datetime"
  });

  belongs_to("subtrack");
  has_many("questions");
}});